kit/inputs/src/MaterialTextView/MaterialTextView.tsx

```diff
@@ -97,6 +99,10 @@ function useExtendedProps(
         onFocus: evt => {
             'worklet';
 
+            if (editable.value === false) {
+                return;
+            }
+
```

Comment:
wait, even though input isn't editable it still fire focus/blur events??? This is weird

Reply:
That behavior is only on web

Reply:
Please add a comment, need to think how to avoid it for mobiles 🤔

Reply:
Or maybe fix this behavior in useTextViewHandler for web.

Reply:
yep, that might work, like for example doesn't return `onFocus` and `onBlur`